rpcs3/Emu/Cell/lv2/sys_timer.cpp

```diff
@@ -13,7 +13,18 @@
 
 LOG_CHANNEL(sys_timer);
 
-void lv2_timer_context::operator()()
+struct lv2_timer_thread
+{
+	shared_mutex mutex;
+	std::deque<std::shared_ptr<lv2_timer>> timers;
+	atomic_t<u32> notify_var = 0;
```

Comment:
I don't think you need a notify var to wake up a thread. May try to issue `operator ()` with empty lambda `[]{}` instead.

Reply:
Fixed.